third_party/2and3/pymysql/connections.pyi

```diff
@@ -94,14 +94,14 @@ class Connection:
     socket: Any
     rfile: Any
     wfile: Any
-    def close(self): ...
+    def close(self) -> None: ...
     def autocommit(self, value): ...
     def commit(self): ...
     def begin(self) -> None: ...
     def rollback(self): ...
     def escape(self, obj): ...
     def literal(self, obj): ...
-    def cursor(self, cursor: Optional[Type[Cursor]] = ...): ...
+    def cursor(self, cursor: Optional[Type[Cursor]] = ...) -> Cursor: ...
```

Comment:
In the future this could be improved to use a type far, but it's good enough for now.